ads/_config.js

```diff
@@ -343,7 +343,16 @@ export const adConfig = {
 
   triplelift: {},
 
-  webediads: {},
+  webediads: {
+    prefetch: [
```

Comment:
Since we only have one prefetch url here, let's not put it in an array. I am not strong about this though @lannka what's your idea on the format here.